mcux/drivers/imx/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ zephyr_library_compile_definitions_ifdef(
   CONFIG_HAS_MCUX_CACHE FSL_SDK_ENABLE_DRIVER_CACHE_CONTROL
 )
 
+zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_FLEXCAN	fsl_flexcan.c)
```

Comment:
The Kinetis `fsl_flexcan.c` needs special compile definitions to avoid compiler warnings (see https://github.com/zephyrproject-rtos/hal_nxp/blob/83e5a13640417a2efb26a31f88b232e4dac31fb2/mcux/drivers/kinetis/CMakeLists.txt#L45). Have you verified that these are not needed for the imx counterpart?

Reply:
I didn't get any compiler warnings when building for `mimxrt1064_evk` or `mimxrt1060_evk`

Reply:
I'll check if the special case for the Kinetis `fsl_flexcan.c` can be removed and follow up if needed.